2-mandatory/1-weather-report.js

```diff
@@ -11,8 +11,14 @@
         - Hint: you can call the temperatureService function from your function
 */
 
+let temp;
 function getTemperatureReport(cities) {
-    // TODO
+    strings = [];
+    cities.forEach(item => {
+        temp = temperatureService(item);
+        strings.unshift(`The temperature in ${item} is ${temp} degrees`);
```

Comment:
It needs to be strings.push on line number 19.

Reply:
You're correct, but why?

Reply:
function getTemperatureReport(cities) {
  let tempArray = [];
  for (i=0; i < cities.length; i++ ) {
    let deg = temperatureService(cities[i]);
    tempArray.push(`The temperature in ${cities[i]} is ${deg} degrees`);
  } 
  return tempArray;
}

It is pushing the string in the array but needs to be in the correct format.